### PR TITLE
Feature GitHub always use token

### PR DESCRIPTION
**Pull request by cordoval**

### CODE REVIEW

**Comment by cordoval** [comment]
@gushphp/contributor for those interested in using the 2fa, i think it is helpful if we say that you can use google authenticator app on your iphone/android, you can download it real quick, then github settings will make you scan a QR code, with that you will have an entry on the app that cycles over verification codes that timeout, when you run gush core:config you will be prompted for one of these codes that time out. After successfully entering it all will be good. It is important to know that if you are locked out, that is the end of your github account. That is why i think disabling it is better.

File: src/ThirdParty/Gitlab/GitLabFactory.php [hunk 1]
```diff
@@ -39,8 +39,8 @@ public function createConfigurator(HelperSet $helperSet, Config $config)
         return new GitlabConfigurator(
             $helperSet->get('question'),
             'GitLab',
-            'http://gitlab-host/api/v3',
-            'http://gitlab-host',
+            'https://gitlab.com/api/v3',
```
Comment (sstok) [inline src/ThirdParty/Gitlab/GitLabFactory.php]: 👍

**Comment by sstok** [comment]
@cordoval Once this activated you can download a list of recovery codes to use in case something goes horrible wrong.

**Comment by cordoval** [comment]
i knew that, and understand now better the 2fa, however even that can be lost or stolen, then you are screwed. 👍

**Pull request merged**